GetandTake/Pages/Error.cshtml.cs

```diff
@@ -22,5 +25,19 @@ public ErrorModel(ILogger<ErrorModel> logger)
     public void OnGet()
     {
         RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier;
+        var exceptionHandlerPathFeature =
+            HttpContext.Features.Get<IExceptionHandlerPathFeature>();
+
+        if (exceptionHandlerPathFeature?.Error is FileNotFoundException)
+        {
+            ExceptionMessage = "The file was not found.";
+            _logger.LogInformation(ExceptionMessage);
```

Comment:
Let's declare all exception message and logs directly in `ExceptionHandler` to prevent logic duplication